### PR TITLE
[TF FE] Handle optional attributes for Convolutional operations

### PR DESCRIPTION
**Pull request by rkazants**
**Details:** TensorFlow convolution operations have two optional attributes with default values: *data_format* and *dilations*. See the specifications for them: [Conv2D](https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/conv2-d), [Conv2DBackpropInput](https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/conv2-d-backprop-input), etc. This fix will top up e2e passrate for DeepLapv3/inception models.

**Ticket:** 88012

Signed-off-by: Kazantsev, Roman <roman.kazantsev@intel.com>

### CODE REVIEW

File: src/frontends/tensorflow/src/op/conv_3d.cpp [hunk 1]
```diff
@@ -17,10 +17,14 @@ namespace op {
...
+
+    // retrieve attributes for Conv3D
     auto tf_strides = node.get_attribute<std::vector<int64_t>>("strides");
-    auto tf_dilations = node.get_attribute<std::vector<int64_t>>("dilations");
     auto tf_padding_type = node.get_attribute<std::string>("padding");
-    auto tf_data_format = node.get_attribute<std::string>("data_format");
+
+    // retrieve optional attributes
+    auto tf_data_format = node.get_attribute<std::string>("data_format", "NDHWC");
+    auto tf_dilations = node.get_attribute<std::vector<int64_t>>("dilations", {1, 1, 1, 1, 1});
```
Comment (mvafin) [inline src/frontends/tensorflow/src/op/conv_3d.cpp:27]: Shouldn't default dilations for 3D case be of size 6, not 5? Same for backprop 3d case

Comment (mvafin) [reply src/frontends/tensorflow/src/op/conv_3d.cpp:26]: Okay, my bad

**Review by mvafin — APPROVED** [review]

**Comment by slyalin** [comment]
I don't see any layer tests in this PR to cover such cases. Do we have them?

**Comment by rkazants** [comment]
> I don't see any layer tests in this PR to cover such cases. Do we have them?

Let me do this in separate PR. Now we have layer tests to cover only Constant K.

**Comment by rkazants** [comment]
> I don't see any layer tests in this PR to cover such cases. Do we have them?

We don't have it in TF1 layer tests to cover Convolutional operations. Also, we don't have mechanism implemented to switch to new frontend. I think these two parts will be done in separate activities. Need to discuss this offline to make sure these tickets exist.

**Pull request merged**